index.html

```diff
@@ -4070,7 +4058,7 @@ <h2>
       </dl>
 
       <p>
-The output variables of this function MUST be as follows:
+The output variables of this function are as follows:
```

Comment:
Why would we not test this normative statement?

Reply:
Eliminating this normative statement has no effect on the number of tests we write because each item in the list asserts a normative test as well and refers to a data structure for which there are already normative tests.